self-update-pipelines/self-update-pipelines_spec.rb

```diff
@@ -31,4 +34,22 @@
     ).to match(/GNU Make 4/)
   end
 
+  it "checks if 'aws' binary exists and is a file" do
+    expect(file(AWSCLI_BIN)).to be_file
+  end
+
+  it "checks if 'aws' binary is executable" do
+    expect(file(AWSCLI_BIN)).to be_mode 755
+  end
+
+  it "has the 'aws' version #{AWSCLI_VERSION}" do
+    def awscli_version
+      command("aws --version").stderr.strip
+    end
+    expect(awscli_version).to match(/aws-cli\/#{AWSCLI_VERSION} /)
+  end
+
+  it "the 'aws help' command works" do
+    expect(command("aws help").stdout).to match(/Optional  parameters/)
```

Comment:
We want to test if `aws help` actually works, because it requires having a functional groff and lesser.

I will pick a different text to check it then.